core/domain/exp_jobs_one_off.py

```diff
@@ -176,3 +178,23 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class FetchAndCalculateExplorationSizeDoFn(beam.DoFn):
```

Comment:
Hi @brianrodri 
I saw there are a couple of issues going on beam jobs, for this issue I am writing a job that adds a new attribute in the ExplorationModel, Is there any issue/pr related to beam job should I wait or am I good to go in writing these beam jobs?

Reply:
added just for a sample, removing from the PR.

Reply:
Also, one more question does loading all the demo exploration and running a job (job, which calculates and update the proto_size_in_bytes) for these demo exploration will be a good idea, or do we need to update these demo exploration files manually?